app/controllers/coronavirus_form/check_answers_controller.rb

```diff
@@ -15,8 +15,13 @@ def show
   def submit
     submission_reference = reference_number
 
-    session[:reference_number] = submission_reference
-    FormResponse.create(form_response: session)
+    session[:reference_id] = submission_reference
```

Comment:
I wonder if we could set this right at the beginning of the journey, then we could use it for logging / Sentry error context. Of course not for this PR :)